wp-content/themes/haruoshi_origin/functions.php

```diff
@@ -189,3 +189,70 @@ function taxonomy_writer() {
 ));
 
 endif;
+
+/**
+ * ページネーション
+ */
+function pagination($pages = 1, $range = 1)
+{
+	$pages = (int)$pages; //float型で渡ってくるので明示的にint型 へ
+	$paged = get_query_var('paged') ? get_query_var('paged') : 1;
+
+	// 1ページしかない時
+	if ($pages === 1) {
+		return;
+	}
+
+	if(1 != $pages) { //全ページが１でない場合はページネーションを表示する
+		// 最初と最後のページにいるときは前後3ページを表示
+		$range = ($paged == 1 || $paged == $pages) ? 2 : 1;
+		
+		echo '<div class="pager">';
+		echo '<ul class="pagination">';
+		if($paged > 1) {
+			echo '<li class="pre"><a href="'.get_pagenum_link($paged - 1).'"><span class="arrow arrow-left"></span></a></li>';
```

Comment:
```suggestion
			echo '<li class="pagination__pre"><a href="'.get_pagenum_link($paged - 1).'"><span class="pagination__arrow pagination__arrow--left"></span></a></li>';
```
こうやって構造化するとSCSSが書きやすくなる